content/en/blog/_posts/2022-08-15-introducing-cosi.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: blog
+title: 'Introducing COSI: Object Storage Support in Kubernetes'
+date: 2022-08-15
+slug: cosi-object-storage-support
+---
+
+**Authors:** Sidhartha Mani (alcion.ai)
+
+This article introduces the Container Object Storage Interface (COSI), a standard for provisioning and consuming object storage in Kubernetes. It is a alpha feature in Kubernetes v1.25.
+
+## Motivation
+
+File and block storage are treated as first class citizens in the Kubernetes ecosystem via CSI. Workloads using CSI volumes enjoy the benefits of portability across vendors and across Kubernetes clusters without the need to change application manifests. An equivalent standard does not exist for Object storage.
+
+Object storage has been rising in popularity in recent years as an alternative form of storage to filesystems and block devices. Object storage paradigm promotes disaggregation of compute and storage. This is done by making data available over the network, rather than locally. Disaggregated architectures allow compute workloads to be stateless, which consequently makes them easier to manage, scale and automate.
+
+## COSI
+ 
+COSI aims to standardize consumption of object storage to provide the following benefits:
+
+* Kubernetes Native - A completely kubernetes-API driven experience to provision, configure and manage buckets
+* Self Service - A clear delineation between administration and operations (DevOps) to enable self-service capability for DevOps personnel
```

Comment:
```suggestion
* Self Service - Allows delegation permission for teams to configure and use object storage, without needing to assign cluster-level administrative access to that team.
```

DevOps is _not_ about clear delineation between administration and operations, and someone following DevOps practices is necessarily working in a pure IT ops function.